packages/components/src/internal/components/samples/SamplesEditableGrid.tsx

```diff
@@ -202,6 +202,11 @@ class SamplesEditableGridBase extends React.Component<Props, State> {
         }
     };
 
+    hasParentDataTypes = (): boolean => {
+        const { parentDataTypes } = this.props;
+        return parentDataTypes?.toArray()?.length > 0;
```

Comment:
minor: this works, but I believe you can avoid the conversion to array with just `parentDataTypes?.size > 0`